appinventor/appengine/src/com/google/appinventor/client/widgets/properties/PropertyHelpWidget.java

```diff
@@ -42,11 +41,11 @@ private PropertyHelpPopup(final EditableProperty prop,
       // Create popup panel.
       super(true);
       setStyleName("ode-ComponentHelpPopup");
-      setTitle(prop.getName());
+      setTitle(ComponentsTranslation.getPropertyName(prop.getName()));
 
       // Create title from component name.
-      Label titleBar = new Label(prop.getName());
-      setTitle(prop.getName());
+      Label titleBar = new Label(ComponentsTranslation.getPropertyName(prop.getName()));
+      setTitle(ComponentsTranslation.getPropertyName(prop.getName()));
       titleBar.setStyleName("ode-ComponentHelpPopup-TitleBar");
 
       // Create content from help string.
```

Comment:
I thought the original reason for the investigation was to show the translated help strings, but there is no adjustment here.

Reply:
Never mind, I see that this was handled in the caller instead.